go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/Bios-Marcel/shortnotforlong v1.1.1
 	github.com/alecthomas/chroma v0.7.3
 	github.com/atotto/clipboard v0.1.2
+	github.com/bunyk/gokeybr v0.0.0-20201019133936-f9e4ed3fdc5d // indirect
```

Comment:
What's this?

Reply:
Installed some things while working on this, thought I had reverted go.sum and go.mod. Will fix